src/components/Wizard/__snapshots__/Wizard.test.js.snap

```diff
@@ -1,822 +1,929 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Wizard embedded renders properly 1`] = `
+exports[`Wizard Pattern Body renders null without steps 1`] = `null`;
+
+exports[`Wizard Pattern renders properly after wizard step click 1`] = `
```

Comment:
these need slimmed down and focused.  I would say as a general practice it is better to keep snapshots under 20 lines if possible under 10.  Any larger and people skim them, and they lose all value. Using shallow would most likely do most of this, and then using selectors would get you more.  I do know that they are currently large, but now would be a good time to update them to add value.